en/docs/guides/Provisioning/outbound-provisioning-with-google.md

```diff
@@ -0,0 +1,148 @@
+# Configure Outbound Provisioning with Google
+
+This topic provides instructions on how to configure Google as the
+Identity Provider to provision users from WSO2 Identity Server. The
+service provider in this scenario is WSO2 Identity Server(WSO2 IS). 
+
+!!! tip "Before you begin!"
+    -   You need to have a Google domain. Click
+        [here](https://www.bettercloud.com/monitor/the-academy/create-google-apps-domain-three-easy-steps/)
+        for more information on creating the domain.
+    
+
+## Configuring Google
```

Comment:
The first heading uses "Configure" but the rest of the headings use "Configuring"...Shall we stick to using "Configure", "Test", "Add" etc instead of using the ing form? 

please check and apply to other headings in this PR